inc/linker.class.php

```diff
@@ -120,15 +156,16 @@ public function linkPostponed() {
       do {
          $postponedCount = 0;
          $postponedAgainCount = 0;
-         foreach ($this->postponed as $itemtype => $postponedItemtypeList) {
+         foreach ($this->postponed as $itemtype => &$postponedItemtypeList) {
             $postponedCount += count($postponedItemtypeList);
-            $newList = $postponedItemtypeList;
+            $newList = [];
             foreach ($postponedItemtypeList as $originalId => $postponedItem) {
                if ($itemtype::import($this, $postponedItem['input'], $postponedItem['relationId']) === false) {
                   $newList[$originalId] = $postponedItem;
                   $postponedAgainCount++;
                }
             }
+            $postponedItemtypeList = $newList;
          }
 
```

Comment:
```suggestion
unset($postponedItemtypeList);
```
Can be a good idea to unset after using a reference in a foreach even if it's not needed in the current code.
This can avoid some nasty bugs if someone decide to add some code later and reuse this var.

See [php doc](https://www.php.net/manual/en/control-structures.foreach.php): `Reference of a $value and the last array element remain even after the foreach loop. It is recommended to destroy it by unset().`

Reply:
+1, i'll add a commit for that